Library/Formula/pcre.rb

```diff
@@ -2,9 +2,11 @@ class Pcre < Formula
   desc "Perl compatible regular expressions library"
   homepage "http://www.pcre.org/"
   url "https://ftp.csx.cam.ac.uk/pub/software/programming/pcre/pcre-8.37.tar.bz2"
-  mirror "https://downloads.sourceforge.net/project/pcre/pcre/8.37/pcre-8.37.tar.bz2"
+  mirror "https://www.mirrorservice.org/sites/downloads.sourceforge.net/p/pc/pcre/pcre/8.37/pcre-8.37.tar.bz2"
```

Comment:
`pcre2` already went [merged](9f746f414f470529e3993de62dfd3ee3b4324fa4) with the same mirror, but I can restore the address, if it is problematic of course.